lib/Backend/BackwardPass.cpp

```diff
@@ -2215,15 +2215,20 @@ BackwardPass::IsLazyBailOutCurrentlyNeeeded(IR::Instr * instr) const
         "liveFixedField is null, MergeSuccBlocksInfo might have not initialized it?"
     );
 
-    if (instr->IsStFldVariant())
```

Comment:
Do you mean to say that we can remove the lazy bailout tag if the StFld is _not_ writing to a live fixed field? If it's writing to a fixed field, that means we may need a lazy bailout even if no implicit call takes place.

Reply:
Have you made the change that we discussed, where we don't look for a lazy bailout record for the instruction if we know it will bail out on implicit calls?

Reply:
good catch.

"...LazyBailOut tag on this instr if the StFld is writing to a live fixed field..." -> "...LazyBailOut tag on this instr if the StFld is writing to a **not** live fixed field..."

Reply:
> where we don't look for a lazy bailout record for the instruction if we know it will bail out on implicit calls?

`BackwardPass::UpdateImplicitCallBailOutKind` does this via `instr->ClearLazyBailOut()`